### PR TITLE
5950 add parameters to mailto links

### PR DESCRIPTION
**Pull request by mekanics**
This PR adds a `subject` and `body` field to the email link in the RichText Editor. 

It does not completely solve #5950 as I didn't add the `cc` & `bcc` fields. The reason for this, is that the standard allows multiple email address in the 'cc' and 'bcc' (actually in the `to` as well – and it should be optional, but that's another story 😉 )

### CODE REVIEW

**Comment by jandor64** [comment]
Will we be seeing this in a version soon? Thanks!

**Comment by daveclarkit** [comment]
+ 1 for getting this into release!

File: wagtail/admin/views/chooser.py [hunk 1]
```diff
@@ -12,6 +12,13 @@
 from wagtail.core.models import Page, UserPagePermissionsProxy
 from wagtail.core.utils import resolve_model_string
 
+try:
+    import urlparse
+    from urllib import urlencode
+except ImportError:  # For Python 3
```
Comment (zerolab) [inline wagtail/admin/views/chooser.py:18]: drive-by comment: we only support Python 3

**Comment by lb-** [comment]
@mekanics sorry this never got a review, are you keen on rebasing/pursuing this still? Would be great to get this up to date and fix the feedback item above.

**Comment by chrismattmann** [comment]
we could really use this in my project at the day job. Any ideas on a workaround until this PR is merged?

**Comment by lb-** [comment]
Superseded by #10920
